app/dashboard/templates/addinterest.html

```diff
@@ -99,6 +95,78 @@ <h5 class="text-center font-title">{% trans "Submit a Plan" %}</h5>
             </div>
             <button id="submit" class="row button button--primary btn-interested" type="submit">{% trans "Next" %}</button>
           </form>
+        {% elif is_registered and bounty.event %}
```

Comment:
so this message appear to people when they are not register yet to the hackathon but now is also showing the form behind. is that right? doesn't people need to go to onboard register and come back?